pandas/tests/series/test_missing.py

```diff
@@ -891,6 +891,16 @@ def test_spline_error(self):
         with tm.assertRaises(ValueError):
             s.interpolate(method='spline', order=0)
 
+    def test_interp_timedelta64(self):
+        # GH 6424
+        tm._skip_if_no_scipy()
```

Comment:
the interpolate method being used over here is 'time', i dont think it requires scipy, i just had this notion that we ignore the interpolation tests if scipy is absent, on a more thorough check i found that is not the case